### ISSUE TITLE
Bad performance on Google Meet

### ISSUE THREAD

**Issue by kgorol** [dr460nf1r3/firedragon-browser#86]
Hi everyone,

Are there any tweaks necessary to make Google Meet work? When I launch meeting, people's avatars don't move at all and are in low quality. Same with people presentation, it looks a bit worse with sporadic refresh.

It is hard to attach any screenshots, as still images look kinda similar to regular Firefox.

**Comment by dr460nf1r3** [comment]
Sorry for the late reply. Did you already try the troubleshooting steps outlined in the README?

**Comment by kgorol** [comment]
@dr460nf1r3 yes, but tbh I am stuck after renaming `firedragon.cfg` because after that step Meet does not work at all. No matter what call I try to join, I get this redirect below:

![image](https://github.com/dr460nf1r3/firedragon-browser/assets/34159482/58d6bb08-360c-4c84-902f-2cc8772cd21f)

**Comment by dr460nf1r3** [comment]
As next troubleshooting steps I'd try whether this also occurs on Floorp itself.
